libs/core/src/scss/interaction-state/_focus.scss

```diff
@@ -14,6 +14,8 @@
 // ---------------------------------------------------------------------------
 @mixin apply-focus-visible($shadow: null, $gap: utils.size('xxxxs')) {
   @include utils.not-touch {
+    @content;
```

Comment:
Moving `@content` to here alters the order in which styles are applied. This may potentially cause the "wrong" styles to be applied, if the `@content` part of where the mixin is applied is supposed to override the default mixin styles.

Does the Sass deprecation notes say anything about this? 

Would a possible solution be to keep `@content` at the bottom of the mixin and wrap it in `& { ... }`?